pyupgrade/_plugins/open_mode.py

```diff
@@ -21,19 +23,35 @@
 U_MODE_REPLACE = U_MODE_REPLACE_R | U_MODE_REMOVE_U
 
 
-def _fix_open_mode(i: int, tokens: List[Token]) -> None:
+class FunctionArg(NamedTuple):
+    arg_idx: int
+    value: ast.expr
+
+
+def _fix_open_mode(i: int, tokens: List[Token], arg_idx: int) -> None:
```

Comment:
let's make `arg_idx` named-only -- this is what I've been doing for other replace functions (makes it less error-prone)